src/metadataGeneration/typeResolver.ts

```diff
@@ -127,7 +130,7 @@ export class TypeResolver {
         additionalType = new TypeResolver(indexSignatureDeclaration.type as ts.TypeNode, this.current, this.parentNode, this.extractEnum, this.context).resolve();
       }
 
-      const objLiteral: Tsoa.ObjectLiteralType = {
+      const objLiteral: Tsoa.NestedObjectMetaType = {
```

Comment:
```suggestion
      const objLiteral: Tsoa.NestedObjectLiteralType = {
```

Please don't change the naming without reason. I agreed to add nested so new contributors can read this more easily. However, TS calls these ObjectLiterals, so everyone familiar with the TS terminology should be respected aswell.

Reply:
Good point